### ISSUE TITLE
Balancing Essense Shift - Nerfing

### ISSUE THREAD

**Issue by ash47** [LegendsOfDota/LegendsOfDota#352]
[This is a discussion]

I think that the reason essence shift is so strong / overpowered is that it is meant to be used on a melee hero. We should consider nerfing it, such that if the target you are attacking is more than melee distance away from you, essence shift acts differently.

Making it so that only every 1st hit in a group of three hits that lands on a target steals attributes, so, if you are attacking, it will STEAL, MISS, MISS, STEAL, MISS MISS, ETC if you are far away. I'll also consider STEAL, STEAL, MISS, STEAL, STEAL, MISS, ETC

Thoughts on making essence shift a more balanced ability on ranged heroes?

**Comment by Snoresville** [comment]
Make silver edge a bit cheaper

**Comment by ash47** [comment]
It might also be interesting to balance picking 3 essence shifts, making it so only one can activate at a time -- meaning, it will apply the bonus to your STR, next might be AGI, and then INT, instead of all three at once, or, in the case of two, it could go STR, AGI, STR, AGI, etc -- making it so you can only steal upto 1 of each attribute per attack.

**Comment by Snoresville** [comment]
maybe essence shift should steal the amount of stats based on attack range so 150 would be 100% and 450 range would be 150/450 = 1/3 of the stats stolen

**Comment by ash47** [comment]
You can only steal whole amounts of stats though, how do you decide which
stats to steal?

**Comment by darklordabc** [comment]
I suggest we make essence shift OP ability but also add a nerfed version as a custom ability.

**Comment by darklordabc** [comment]
Then for the custom abilities, we could add a 1 second cooldown thats dependant on how far away you are. So less than 200, no cooldown, but every 100 over that adds like a .25 second cooldown. So at 600 it will be a 1 second cooldown.

**Comment by darklordabc** [comment]
I don't like the idea of changing the default dota skills just in case people want to use the originals in custom gamemodes.

**Comment by Snoresville** [comment]
we should ship agi and int flesh heap so then duelists can get the upper hand to stat stealers

**Comment by Jexah** [comment]
I think generally, the idea about putting any modified skills in the custom tab and leaving the default ones in the normal tab as banned is the right way to go.

Instead of basing it off of attack speed (hit, miss, miss), maybe just have it apply a cooldown on ranged hero, independent of their range.

By the way, Snoresville, Flesh Heap doesn't steal stats.

**Comment by Snoresville** [comment]
exactly

**Comment by Snoresville** [comment]
we need to have people stacked with flesh heap so they wont have to really worry about stats

**Comment by Jexah** [comment]
Can you please not reply with 1-word messages, and if you were the last person to post, please just edit your original message to include the new message. We all get emails every time you post.

People will ALWAYS worry about stats. That is completely illogical thinking.

**Comment by Snoresville** [comment]
i guess i think im wrong in a sense.
but we still have silver edge, i think we should have something with break that could combat against essence shift spammers

**Comment by iprobie** [comment]
why don't you just add it to banlist just like how you put Grow, Glaives of Wisdom, Fiery Soul and Fury Swipes?

**Comment by darklordabc** [comment]
A melee hero only condition for abilities would fix this fury swipes and grow.

**Comment by Jexah** [comment]
Wouldn't fix Grow, but would fix Fury Swipes.

**Comment by darklordabc** [comment]
Grow on melee wouldnt be OP tho.

**Comment by Jexah** [comment]
You're wrong, I don't have time right now to explain.

**Comment by Snoresville** [comment]
or maybe we could go by the idea of "build your own overpowered masterpiece" and not nerf it at all
and then go further by building an overpowered masterpiece to counter it

**Comment by darklordabc** [comment]
With that logic nothing should be banned.

**Comment by Snoresville** [comment]
well yeah might as well bring back reincarnation and borrowed time to counter the stupid essence shifts

**Comment by iprobie** [comment]
what are you talking about? people with essence shifts LOVE playing against borrowed time and reincarnation

**Comment by Snoresville** [comment]
based on the assumption that the ones with essence shifts can actually hit you and not be focused down with stuns, bashes or otherwise, i would say yes.
but let's not think that essence shift is so powerful that people can throw it into their builds and win the whole game. you have four other players who will for the most part help you and the team to win. also there are ways to counter essence shift players (probably not reincarnation and borrowed time) such as stuns, decrepify, illusions and a couple something more items such as heaven's halberd and silver edge. i think that essence shift should not be nerfed and instead we should think a bit further on how to counter it, like 5 man walrus kicks

**Comment by ryanwilson112** [comment]
Sorry for the late feedback, but i'm against nerfing it. 

Essence shift is a fun ability, but i'm not sure it's so op that it's 100% core on ranged carry builds. Let alone is it good enough to justify taking all 3 essence shifts.

When you think of Slark, you think of fast attacks, draining the enemy stats to fight for 2 minutes if not killed. It's not a hard attack. That's essentially the inherent problem with essence shift, it's not a burst ability. Therefore it is easily countered by maneuverability. 

If you nerf essence shift, you're just going to shift more people over to spamming pl illusions instead. Which, by the way, is also designed for a melee hero and arguably pretty op too. Spawning an illusion on hit, that instantly reacts by shooting whoever or whatever is in his much larger range, allows for more illusions faster as they're not restricted by angles of attack. 

I've never, ever thought to myself "this ability is op, it needs nerfed". I've played numerous games with a streamer and it's almost always picked, but it's super weak against literally any burst / gank. It is also super, super reliant on a good skill build to go with it. It's got to be said, my chrono 3 essence shifts wasn't performing when I decided to change it up a bit for the banter. And likewise, if you're building yourself to be a carry with damage you're gonna die too fast to get shifts or kill too fast to get shifts, both of which make the ability weak. Which we see many new players do in an attempt to replicate good essence shift games they've seen. 

It's so 50/50. I don't think we punish the good players who know how to the use the ability efficiently.

**Comment by LODonly** [comment]
In my opinion it's better to make a troll combo using 2-3 different essense shifts. Before adding str and int versions of this skill it was ballanced.
I'm against making this skill OP because it's one of anti tank skills. Also illusion builds or burst build counter essense shift builds.

**Comment by Jexah** [comment]
I don't think 3x Essence Shift is OP. This topic is really a
work-in-progress.

**Comment by ryanwilson112** [comment]
I'm against making 2 / 3 of them a troll combo also. 
Most definitely doesn't deserve a spot amongst the infinite mana / stuns or instant one-shot combos. 
Way, way too many weaknesses inherent with it.

Okay, so i've been thinking, do you want to essentially change the essence shift's purpose away from ranged carry? Right now it's used to punish long engagements and encourages short fights, you escape quickly against essence shifts, you don't want to hang around because he's going to hit super hard, fast and you're gonna die fast.
 
if you want to do Hit/Miss/Miss as you suggested, it kills the ability **entirely** in that regard for ranged heroes. You stick around longer because they're not getting insanely hard-hitting and you're keeping your stats to punish them. 

So you need to buff it elsewhere, the duration. 15/30/60/120 should be increased to 30/60/120/240. 
It's essentially no longer a carry ability but a harass one? 4 minutes of discouraged combat. But the stolen stats is so poor it might still be so insanely weak....

So I would want to know whether or not you can dig deeper into the ability and make only the stat gain or stat loss hit/hit/miss. For example. 

Stat gain = Hit / Miss / Miss
Stat loss = Hit / Hit / Miss. 

As this builds into the ranged harass more than it does carry....
I think this could be acceptable....but i'm not sure at all. It's kinda...clanky?

**Comment by Snoresville** [comment]
just keep it as it is and focus on making builds that kill the essence shifters instantly
